local config = {
	DEFAULT_PLAYER_DATA = {
		towers = 0,
	},
	DATA_STORE_KEY = "Production",
	TOWER_STAT_NAME = "Towers",

	TIMER_SYNC_INTERVAL = 2,
	CHECKPOINT_INTERVAL = 0.25,
	RESTART_COOLDOWN = 3,
	RESTART_RESETS_TOWER_RUSH = true,

	WINPAD_INTERVAL = 0.25,
	WINPADS_CHANGE_COLOR = true,
	WINPADS_CHANGE_MATERIAL = true,
	USE_DISPLAY_NAME = false,

	CHECKPOINTS_WORKSPACE_WARNING = true,
	CHECKPOINTS_MISSING_WARNING = true,
	NO_MINIMUM_TIME_WARNING = true,
	R15_WARNING = true,
}

export type PlayerData = typeof(config.DEFAULT_PLAYER_DATA)

return config
